### PR TITLE
Added test cases

### PR DESCRIPTION
**Pull request by ksr30**

### CODE REVIEW

File: src/tests.rs [hunk 1]
```diff
@@ -1,118 +1,321 @@
...
+    let system: u64 = 0;
+
+    let asset_info: AssetInfo<Test> =  AssetInfo{
+        total_issuance: 200u128,
+        issuer: system,
+        permissions: Permissions::SystemLevel,
+        existential_deposit: 1u128
+    };
+    let alice: u64 = 1;
+    let native_asset =  ("Native").using_encoded(<Test as frame_system::Trait>::Hashing::hash);
```
Comment (Gauthamastro) [inline src/tests.rs:23]: Why did you recreate native_asset here?

File: src/tests.rs [hunk 2]
```diff
@@ -1,118 +1,321 @@
...
+        setup_native();
+        let alice: u64 = 1;
+        assert_eq!(Native::burn(50u128), PositiveImbalance::new(50u128));
+    });
+
+    // Burn amount more then total issuance
+    new_test_ext().execute_with(|| {
+        setup_native();
+        let alice: u64 = 1;
+        assert_eq!(Native::burn(250u128), PositiveImbalance::new(200u128)); //@Gautham check this
```
Comment (Gauthamastro) [inline src/tests.rs:93]: This is right!

File: src/tests.rs (hunk 1 above)
Comment (Gauthamastro) [inline src/tests.rs:23]: Why did you initialize native_asset again?

File: src/tests.rs (hunk 2 above)
Comment (Gauthamastro) [inline src/tests.rs:93]: This is all right!

**Review by Gauthamastro — CHANGES_REQUESTED** [review]

File: src/tests.rs (hunk 1 above)
Comment (ksr30) [reply src/tests.rs:23]: oops ... let me remove that

**Comment by ksr30** [comment]
[DEX-202.08 [DEV] Test Custom Asset pallet](https://trello.com/c/kvRbZAbt/37-dex-20208-dev-test-custom-asset-pallet)

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**